modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/gnirs/Gnirs.scala

```diff
@@ -132,14 +135,14 @@ object Gnirs {
 
 
   private def autoFilter(w: Length): GnirsController.Filter2 = {
-    val table = Map(
+    val table = List(
       GnirsController.Filter2.X -> 1.17,
       GnirsController.Filter2.J -> 1.42,
       GnirsController.Filter2.H -> 1.86,
-      GnirsController.Filter2.J -> 2.70,
+      GnirsController.Filter2.K -> 2.70,
       GnirsController.Filter2.L -> 4.30,
       GnirsController.Filter2.M -> 6.0
-    ).mapValues(_.nanometers)
+    ).map{ case (f, w) => (f, w.nanometers) }
 
     table.foldRight[GnirsController.Filter2](GnirsController.Filter2.XD){ case (t, v) => if(w < t._2) t._1 else v}
```

Comment:
Why do you need to do this?

Reply:
I don't need it, I just wanted to have everything as the proper physical quantity, but was too lazy as to add the `.nanometers` to each value.